### ISSUE TITLE
I Can't run command "seeds serve" or "seeds generate [name]"

### ISSUE THREAD

**Issue by goalkeeper112** [terminalvelocity/seeds.js#5]
I've created my seeds app, but when i write on my terminal any commands of seeds-cli, the cli throw the next error:
/usr/local/lib/node_modules/seeds-cli/node_modules/soil-cli/lib/cli.js:60
    return this.commands[this.args[0]](this);
                                      ^
TypeError: this.commands[this.args[0]] is not a function
    at SeedsCLI.command (/usr/local/lib/node_modules/seeds-cli/node_modules/soil-cli/lib/cli.js:60:39)
    at /usr/local/lib/node_modules/seeds-cli/bin/seeds:21:7
    at /usr/local/lib/node_modules/seeds-cli/node_modules/resolve/lib/async.js:46:14
    at process (/usr/local/lib/node_modules/seeds-cli/node_modules/resolve/lib/async.js:173:43)
    at ondir (/usr/local/lib/node_modules/seeds-cli/node_modules/resolve/lib/async.js:188:17)
    at load (/usr/local/lib/node_modules/seeds-cli/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/local/lib/node_modules/seeds-cli/node_modules/resolve/lib/async.js:92:31)
    at /usr/local/lib/node_modules/seeds-cli/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:78:15)

I've installed Io.js V2.1.0 and the last version of the seeds-cli

**Comment by jhliberty** [comment]
The package was under the name `seeds-cli`.. about halfway through I contacted the person who had the name `seeds` and since he wasn't using it he graciously gave it to the project. 

`npm uninstall -g seeds-cli` then `npm i -g seeds` and that should get ya in business. 

I unpublished the seeds-cli package since I have a feeling this would happen frequently if I didn't.

Let me know if that works for ya - Thanks for posting your issue and using seeds!
